pkg/events/trigger/context.go

```diff
@@ -74,6 +74,7 @@ func (store *context) Apply(event trace.Event) (trace.Event, error) {
 	invoking.EventID = event.EventID
 	invoking.ReturnValue = 0
 	invoking.Args = make([]trace.Argument, len(event.Args))
+	invoking.MatchedScopes = event.MatchedScopes
```

Comment:
Add also the `timestamp` as for now it is not available but expected

Reply:
Initializing it breaks unit tests. We'll tackle this after.